### PR TITLE
Fix: add mobile app association files

### PR DESCRIPTION
**Pull request by katspaugh**
They are needed for the mobile apps.

### CODE REVIEW

**Review by iamacook — APPROVED** [review]

File: public/.well-known/assetlinks.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "relation": ["delegate_permission/common.handle_all_urls"],
+    "target": {
+      "namespace": "android_app",
+      "package_name": "io.gnosis.instantsafe",
```
Comment (elgatovital) [inline public/.well-known/assetlinks.json:6]: this package name is incorrect. Please use io.gnosis.safe for production

File: public/.well-known/assetlinks.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      "sha256_cert_fingerprints": [
+        "BE:40:A7:47:33:CA:5B:B6:93:9A:AE:CE:12:11:B0:0F:E1:BC:EA:6C:6C:E6:B6:03:CF:0D:28:FF:46:AE:46:1C"
+      ]
+    }
+  }
+]
```
Comment (elgatovital) [inline public/.well-known/assetlinks.json:12]: also please add entry for internal version
```json
{
    "relation": ["delegate_permission/common.handle_all_urls"],
    "target": {
      "namespace": "android_app",
      "package_name": "io.gnosis.safe.internal",
      "sha256_cert_fingerprints": [
        "a8:de:8a:80:10:e9:aa:9d:80:7a:d6:4a:0c:d9:c4:62:11:27:01:ed"
      ]
    }
  }
```

Comment (elgatovital) [reply public/.well-known/assetlinks.json:12]: and entry for debug 
```json
{
    "relation": ["delegate_permission/common.handle_all_urls"],
    "target": {
      "namespace": "android_app",
      "package_name": "io.gnosis.safe.debug",
      "sha256_cert_fingerprints": [
        "a8:de:8a:80:10:e9:aa:9d:80:7a:d6:4a:0c:d9:c4:62:11:27:01:ed"
      ]
    }
  }
```

Comment (katspaugh) [reply public/.well-known/assetlinks.json:12]: Thanks Vitaly! Could you check again? Not sure I pasted it in the right place.

Comment (elgatovital) [reply public/.well-known/assetlinks.json:12]: My comments were related only to assetlinks.json that is used by the android app. Each app association is represented by an element in array. (see https://developer.android.com/training/app-links/verify-android-applinks#multiple-apps)

**Comment by biafra23** [comment]
I don't know about Android. 

But for iOS this files needs to be accessible at the same hostname that is used for the deep links. As far as I can tell that is 

app.safe.global

So we need this file at: https://app.safe.global/.weel-known/apple-app-site-association

And that probably means: not in this repository?

File: public/.well-known/assetlinks.json (hunk 2 above)
Comment (elgatovital) [reply public/.well-known/assetlinks.json:12]: also we might need to put this file under app.safe.global (see https://developer.android.com/training/app-links/verify-android-applinks#multi-host)

**Comment by katspaugh** [comment]
I see, @biafra23, closing this then.

Please provide the complete and correct files for app.safe.global and I will upload them.

**Comment by katspaugh** [comment]
Alternatively please feel free to make the PR yourself in https://github.com/safe-global/web-core, the files should be in the root of the [`public`](https://github.com/safe-global/web-core/tree/dev/public) folder.

**Comment by biafra23** [comment]
@katspaugh I am sorry. I was wrong. Given the correct entitlement (`*.safe.global` instead of `safe.global`) in the iOS app, it is possible to have this file at: https://safe.global/.weel-known/apple-app-site-association

**Pull request closed without merging**